back-end/controllers/fixtures_byId.controller.js

```diff
@@ -0,0 +1,15 @@
+const { fetchFixturesById } = require("../models/fixtures_byId.model.js");
+
+const getFixturesById = (req, res, next) => {
+  const { fixture_id } = req.params;
+  fetchFixturesById(fixture_id)
+    .then((fixture) => res.status(200).send({ fixture }))
+    .catch((err) => {
+      if (err.status && err.message) {
```

Comment:
ditto the above as we capture the error in app.js

Reply:
amended